transform/mattermost-analytics/seeds/features/event_to_feature_mapping.csv

```diff
@@ -0,0 +1,5 @@
+event_name,category,event_type,feature_name
+event,api,api_posts_ack,Message Acknowledgement (Prof)
```

Comment:
Think you can remove the "(Prof)" (here and below). We can add it back as a separate column later.

Reply:
Will remove it. It's not used anywhere right now.